README.md

```diff
@@ -70,6 +72,18 @@ cd bsn &&
 mon launch bsn.launch
 ``` 
 
+If you want to coneect with Arduino and real sensors use:
+
+```
+roscore
+rosrun rosserial_python serial_node.py "Arduino port address"
```

Comment:
What does "Arduino port address" means? Can you give some intuition of the format of the Arduino port address to minimise confusions here? (e.g. 0.0.0.0)